app/views/request.py

```diff
@@ -26,28 +45,57 @@ def post(self, request):
 
 
 class RequestDetail(APIView):
+    """
+    Request details
+        :param APIView:
+    """
+
     permission_classes = (IsAuthenticated,)
 
-    def get_object(self, pk):
+    # pylint: disable=invalid-name
+    def get_object(self, pk: str) -> Request:
+        """
+        Get single request
+            :param pk: request primary key
+            :return: request
+        """
         try:
             return Request.objects.get(id=pk)
-        except Request.DoesNotExist:
-            raise Http404
+        except Request.DoesNotExist as err:
+            raise Http404 from err
 
-    def get(self, request, pk):
+    def get(self, request: Request, pk: str) -> Response:
```

Comment:
If pylint throws an error for `pk` consider changing the argument name to something clearer